packages/presentation/lib/screen/main_my_bloc.dart

```diff
@@ -55,34 +56,43 @@ class MainBlocImpl extends BlocImpl implements MainBloc {
   }
 
   @override
-  void generateRandomNumber() {
-    _screenData.randomNumber = _generateNumberUseCase();
-    _screenData.isDisabled = false;
+  void startNewGame() {
+    _screenData = _screenData.copyWith(
+      randomNumber: _generateNumberUseCase(),
```

Comment:
extract `_generateNumberUseCase` result to separate variable and then pass it into `copyWith`